### PR TITLE
feat: add status and recent action columns to assigned table

### PR DESCRIPTION
**Pull request by adamstankiewicz**
Ticket: [ENT-7810](https://2u-internal.atlassian.net/browse/ENT-7810)

image

image

image

image

image

# For all changes

- [x] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [x] Ensure to attach screenshots
- [x] Ensure to have UX team confirm screenshots

### CODE REVIEW

File: src/components/learner-credit-management/assignments-status-chips/WaitingForLearner.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+            </p>
+          </div>
+        </BaseModalPopup.Content>
+      </BaseModalPopup>
+    </>
+  );
+};
+
+WaitingForLearner.propTypes = {
+  learnerEmail: PropTypes.string,
```
Comment (iloveagent57) [inline src/components/learner-credit-management/assignments-status-chips/WaitingForLearner.jsx:52]: nit: isRequired?

File: src/components/learner-credit-management/assignments-status-chips/NotifyingLearner.jsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            confirm the assignment notification was successful.
+          </p>
+        </BaseModalPopup.Content>
+      </BaseModalPopup>
+    </>
+  );
+};
+
+NotifyingLearner.propTypes = {
+  learnerEmail: PropTypes.string,
```
Comment (iloveagent57) [inline src/components/learner-credit-management/assignments-status-chips/NotifyingLearner.jsx:42]: nit: isRequired?  Assignment responses should always have an associated email.

**Review by iloveagent57 — APPROVED** [review]

File: src/components/learner-credit-management/assignments-status-chips/NotifyingLearner.jsx (hunk 2 above)
Comment (adamstankiewicz) [reply src/components/learner-credit-management/assignments-status-chips/NotifyingLearner.jsx:42]: With the GDPR/PII pipeline eventually getting integrated for assignments (https://2u-internal.atlassian.net/browse/ENT-7557), I believe there will eventually no longer be a guarantee that `learner_email` will always be present if/when PII data gets scrubbed for assignments. Being proactive and handling those situations in the UI now similar to how we have conditional logic for PII scrubbing in the "Spent" table powered by the analytics API today (i.e., the table cell says "Email hidden" with a tooltip explaining why).

Comment (iloveagent57) [reply src/components/learner-credit-management/assignments-status-chips/NotifyingLearner.jsx:42]: ah good point

**Pull request merged**